cax/cax.json

```diff
@@ -3,76 +3,78 @@
     "name": "xenon1t-daq",
```

Comment:
Possible to change this to xe1t-datamanager instead? See gridcopy branch where gfal-copy pushing from xe1t-datamanager so there should be a local DB entry there